<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_elly">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>HY궁서B</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>엘리멘탈</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_winter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>HY궁서B</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>겨을왕국</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_rosy">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>HY궁서B</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>러브로지</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_out">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>HY궁서B</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>인사이드아웃</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="elly">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>521</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Users/test2/Desktop/elly3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_winter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>551</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Users/test2/Desktop/winter2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_elly">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>551</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rosy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>551</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Users/test2/Desktop/rosy.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_out">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>551</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Users/test2/Desktop/out.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
